pycm/pycm_output.py

```diff
@@ -15,14 +15,7 @@ def html_init(name):
     :type name : str
     :return: html_init as str
     """
-    result = ""
-    result += "<html>\n"
-    result += "<head>\n"
-    result += "<title>" + str(name) + "</title>\n"
-    result += "</head>\n"
-    result += "<body>\n"
-    result += '<h1 style="border-bottom:1px solid ' \
-              'black;text-align:center;">PyCM Report</h1>'
+    result = HTML_INIT_TEMPLATE.format(str(name), OG_DESCRIPTION, OG_IMAGE_URL)
     return result
```

Comment:
You can also return it directly, like
```
return HTML_INIT_TEMPLATE.format(str(name), OG_DESCRIPTION, OG_IMAGE_URL)
```

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/368/commits/fb72bbf1568c129a5e48284386c2aa66bc48ac1f